libsolidity/analysis/TypeChecker.cpp

```diff
@@ -609,29 +609,6 @@ bool TypeChecker::visit(VariableDeclaration const& _variable)
 		}
 	}
 
-	if (varType->dataStoredIn(DataLocation::Storage))
-	{
-		auto collisionMessage = [&](string const& variableOrType, bool isVariable) -> string {
-			return
-				(isVariable ? "Variable " : "Type ") +
-				util::escapeAndQuoteString(variableOrType) +
```

Comment:
I wonder why it was being escaped. Apart, from newlines, I don't think there's anything that could break a comment. I thought that anything past the second `:` was just free-form text.

Reply:
Maybe "escapeAndQuoteString" was used for its "quote" part of it, not for "escape".

Reply:
Might be.